modules/debts_api/spec/lib/debt_api/v0/financial_status_report_service_spec.rb

```diff
@@ -182,31 +182,90 @@ def mock_pdf_fill
   end
 
   describe '#submit_vha_fsr' do
-    let(:form_submission) { build(:debts_api_form5655_submission) }
-    let(:user) { build(:user, :loa3) }
+    let(:user_account) { create(:user_account) }
```

Comment:
Consider trying `build_stubbed` instead of `create` it will give you an `id` without hitting the db